test/powershell/Language/Classes/Scripting.Classes.Attributes.Tests.ps1

```diff
@@ -292,7 +292,12 @@ Describe 'ValidateSet support a dynamically generated set' -Tag "CI" {
 
             $testAssemply = "$TestDrive\tst-$(New-Guid).dll"
             Add-Type -TypeDefinition $a -OutputAssembly $testAssemply
-            Import-Module $testAssemply
+            $testModule = Import-Module $testAssemply -PassThru
+        }
+
+        AfterAll {
+            Remove-Module -ModuleInfo $testModule
+            Remove-Item $testAssemply -Force -ErrorAction SilentlyContinue

```

Comment:
We can remove the line - the Issue (get-help) should be resolved by unloading the test module